docs/GeneratorDeveloperGuide.md

```diff
@@ -0,0 +1,11 @@
+## Key Concepts:
+
+1. [Design Decisions](KeyDecisions.md)
+1. [Decision Trees](DecisionTrees/DecisionTrees.md)
+1. [Profile Syntax](ProfileGrammar.md)
+
+
+## Development
+
```

Comment:
maybe add:

`1. Development Environment Setup`

`1. Creating a Pull Request`

`1. Raising an Issue on Github`

???

Reply:
we have a contributing.md 
we can just link to that instead of adding these 3 i believe

Reply:
That file seems to contain all those items. I've added it to the list of linked files.